src/CLI.java

```diff
@@ -15,10 +15,11 @@ public static void main(String[] args) {
                             "[1] Login as student\n" +
                             "[2] Login as teacher\n" +
                             "[3] Register as new student\n" +
-                            "[4] Search for ID by phone nubmer\n" +
-                            "[5] Login as admin"
+                            "[4] Search for ID by phone number\n" +
+                            "[5] Login as admin\n" +
+                            "[6] Retrieve all section information by Class ID"
```

Comment:
We don't want people who aren't logged in to get class information, so we shouldn't include this in the login screen. You can simply remove this option, as well as options 4 and 5 to fix the merge conflict.